ios/RNZoomUs.m

```diff
@@ -52,22 +51,26 @@ - (dispatch_queue_t)methodQueue
     initializePromiseResolve = resolve;
     initializePromiseReject = reject;
 
+
+
     MobileRTCSDKInitContext *context = [[MobileRTCSDKInitContext alloc] init];
-    context.domain = webDomain;
+    context.domain = data[@"domain"];;
     context.enableLog = YES;
     context.locale = MobileRTC_ZoomLocale_Default;
 
     //Note: This step is optional, Method is uesd for iOS Replaykit Screen share integration,if not,just ignore this step.
     // context.appGroupId = @"group.zoom.us.MobileRTCSampleExtensionReplayKit";
     BOOL initializeSuc = [[MobileRTC sharedRTC] initialize:context];
+    [[[MobileRTC sharedRTC] getMeetingSettings]
+      disableShowVideoPreviewWhenJoinMeeting:settings[@"disableShowVideoPreviewWhenJoinMeeting"]];
```

Comment:
is there a method like that for android and could we use it?

Reply:
Not sure. Also look like it is temp solution, so probably will be removed in future SDK updates.
Check it: https://github.com/mieszko4/react-native-zoom-us/issues/28